src/main/java/com/bric/awt/BristleStroke.java

```diff
@@ -136,86 +137,90 @@ public Shape createStrokedShape(Shape p) {
         MeasuredShape[] paths = MeasuredShape.getSubpaths(p);
 
         for (int a = 0; a < layers; a++) {
-            float k1 = ((float) a) / ((float) (layers - 1));
-            float k2 = (k1 - 0.5f) * 2; //range from [-1,1]
+            drawStrokedShapes(path, r, paths, a);
+        }
+        return path;
+    }
 
-            float k3 = thickness;
-            float minGapDistance = (4 + 10 * k3) / (1 + 9 * spacing);
-            float maxGapDistance = (40 + 10 * k3) / (1 + 9 * spacing);
-            float dd = minGapDistance * (1 + 20 * (1 - thickness) * Math.abs(k2 * k2));
+    private void drawStrokedShapes(GeneralPath path, Random r, MeasuredShape[] paths, int a) {
```

Comment:
Extracting this method might make sense, but as I said, I'm unwilling to make changes outside the pixelitor package, unless really necessary. I don't even fully understand what is going on here, and I don't want to spend time with understanding it...